x-pack/plugins/security_solution/public/detection_engine/rule_management/logic/bulk_actions/translations.ts

```diff
@@ -48,10 +55,27 @@ export function explainBulkSuccess(action: BulkActionType, summary: BulkActionSu
 
     case BulkActionType.disable:
       return i18n.RULES_BULK_DISABLE_SUCCESS_DESCRIPTION(summary.succeeded);
+  }
+}
 
-    case BulkActionType.edit:
-      return i18n.RULES_BULK_EDIT_SUCCESS_DESCRIPTION(summary.succeeded);
+export function explainBulkEditSuccess(
+  editPayload: BulkActionEditPayload[],
+  summary: BulkActionSummary
+): string {
+  if (
+    editPayload.some(
+      (x) =>
+        x.type === BulkActionEditType.add_index_patterns ||
+        x.type === BulkActionEditType.set_index_patterns ||
+        x.type === BulkActionEditType.delete_index_patterns
+    )
+  ) {
+    return `${i18n.RULES_BULK_EDIT_SUCCESS_DESCRIPTION(summary.succeeded)}. ${
+      i18n.RULES_BULK_EDIT_SUCCESS_INDEX_EDIT_DESCRIPTION
+    }`;
```

Comment:
Will modify this logic in https://github.com/elastic/kibana/pull/144461 so that the warning that explains that rules with data views were skipped will only show when rules were actually skipped, along with the detail of how many rules were skipped.